### PR TITLE
[DO NOT MERGE] Issues/cif 1444

### PR DESCRIPTION
**Pull request by dplaton**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

File: react-components/src/components/Account/account.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+import React from 'react';
+
+import AccountTrigger from '../AccountTrigger';
+import AccountDropdown from '../AccountDropdown';
+
+const Account = () => {
```
Comment (dplaton) [inline react-components/src/components/Account/account.js:19]: We should rename this to `AccountContainer` or something similar.

File: react-components/src/components/AccountTrigger/accountTrigger.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        stroke: iconColor
+    };
+
+    const button = (
+        <button
+            className={classes.root}
+            aria-label="Toggle account dropdown"
+            onClick={() => toggleAccountDropdown(!isAccountDropdownOpen)}>
+            <Icon src={UserIcon} attrs={svgAttributes} />
+            <AccountIconText />
```
Comment (dplaton) [inline react-components/src/components/AccountTrigger/accountTrigger.js:41]: Can we merge the `AccountIconText` component here? I imagine we could have one component that would show the label (or not) based on a prop, i.e. `<AccountTrigger label="my optional text here"/>`.

File: react-components/src/components/Account/account.js (hunk 1 above)
Comment (nettree) [reply react-components/src/components/Account/account.js:19]: Sure, no problem, shall I rename the component name I added to `ui.apps`?

File: react-components/src/components/AccountTrigger/accountTrigger.js (hunk 2 above)
Comment (nettree) [reply react-components/src/components/AccountTrigger/accountTrigger.js:41]: Sure, will have a look and let you know when it's done.

File: react-components/src/components/Account/account.js (hunk 1 above)
Comment (dplaton) [reply react-components/src/components/Account/account.js:19]: No, the AEM component can keep its name

Comment (nettree) [reply react-components/src/components/Account/account.js]: Hi @dplaton, the `<Account>` component has been renamed to `<AccountContainer>`

File: react-components/src/components/AccountTrigger/accountTrigger.js (hunk 2 above)
Comment (nettree) [reply react-components/src/components/AccountTrigger/accountTrigger.js]: Hi @dplaton, `<AccountTrigger>` will now accept a label as a prop and render it if it's not undefined

**Comment by nettree** [comment]
Rename branch `issues/CIF-1444` to `CIF-1444`

**Comment by nettree** [comment]
Will create a new PR based on new branch `CIF-1444`

**Pull request closed without merging**